contracts/src/lib/LibGateway.sol

```diff
@@ -442,20 +444,22 @@ library LibGateway {
     /// failing network.
     /// (we could optionally trigger a receipt from `Transfer`s to, but without
     /// multi-level execution it would be adding unnecessary overhead).
-    function sendReceipt(IpcEnvelope memory crossMsg, OutcomeType outcomeType, bytes memory ret) internal {
-        if (crossMsg.isEmpty()) {
-            revert CannotCreateIpcReceipt();
+    function sendReceipt(IpcEnvelope memory original, OutcomeType outcomeType, bytes memory ret) internal {
+        if (original.isEmpty()) {
+            // This should not happen as previous validation should prevent empty messages arriving here.
+            // If it does, we simply ignore.
+            return;
```

Comment:
This diff is dirty as this change is already part of main. Let's try to rebase if possible.

Reply:
Will do